cost-analyzer/templates/kubecost-cluster-controller-api-secret-template.yaml

```
@@ -0,0 +1,15 @@
+{{- if and .Values.clusterController.enabled .Values.clusterController.createPrimaryApiSecret }}
+{{- if and .Values.clusterController.createPrimaryApiSecret (lt (len .Values.clusterController.primaryApiKey) 5) }}
+{{- fail "\n\nWhen clusterController.createPrimaryApiSecret is true, primaryApiKey must be set." }}
```

Updating the error message here to be in line with the logic

```suggestion
{{- fail "\n\nWhen clusterController.createPrimaryApiSecret is true, primaryApiKey must be set and longer than 5 characters" }}
```